tests/sensors/omron_d6t/test_omron_d6t.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from mock import patch, Mock
+from sensors.sensor import Sensor
+from sensors.omron_d6t.omron_d6t import OmronD6T
+
+class TestOmronD6T(TestCase):
+    def setUp(self):
+        with patch.object(Sensor, "__init__", return_value = Mock()):
+            self.__omron_d6t = OmronD6T()
+
+        self.__room_temperature = 25
+        self.__temperature_values = [ 25.5, 22.5, 28.9, 26.2,
+                                    22.4, 22.2, 29.4, 29.9,
+                                    28.4, 23. , 28.0, 27.7,
+                                    22.9, 24.3, 28.8, 27.6 ]
+
+    def test_buffer_length(self):
+        self.assertEqual(35, self.__omron_d6t.buffer_length)
+
+    @patch("sensors.omron_d6t.omron_d6t.OmronD6T._OmronD6T__read")
```

Comment:
By doing this, we patch `OmronD6T.__read` with calls for raw data from `Sensor`.

@MikeVezina could you send me a snippet of the raw data coming from the sensor? As in the byte array so that I can add more tests.

Reply:
I think it would be good if we can rely completely on pigpio for i2c and gpio operations.

pigpio provides remote connections to pi's (good for remote debugging), but also, relying solely on pigpio would make it much easier for us, as we'd only have to mock the methods provided by pigpio.

Thoughts?

Reply:
Sent over messenger!